pkg/storage/stores/shipper/indexshipper/boltdb/compactor/compacted_index.go

```diff
@@ -178,7 +178,7 @@ func (c *CompactedIndex) ToIndexFile() (shipperindex.Index, error) {
 	if c.compactedFileRecreated {
 		fileNameFormat = "%s" + recreatedCompactedDBSuffix
 	}
-	fileName := fmt.Sprintf(fileNameFormat, shipperutil.BuildIndexFileName(c.tableName, uploaderName, fmt.Sprint(time.Now().Unix())))
+	fileName := fmt.Sprintf(fileNameFormat, shipperutil.BuildIndexFileName(c.tableName, uploaderName, fmt.Sprint(time.Now().UnixNano())))
```

Comment:
wondering if this would break reading any old files with still `seconds` precision instead of `nanoseconds`?

Reply:
It should not, we do not care about filename format in boltdb-shipper.